src/features/FeatureExtractor.js

```diff
@@ -294,7 +294,9 @@ class FeatureExtractor {
     _handleConfStartTime = dumpLineObj => {
         const [ , , timestamp ] = dumpLineObj;
 
-        this.conferenceStartTime = timestamp;
+        // At the end of a conference jitsi-meet sends another `conferenceStartTime` event
+        // with 0 as the start time, so we ignore it.
+        timestamp && (this.conferenceStartTime = timestamp);
```

Comment:
Do we know why this happens? Sounds like a bug in the meet?

Reply:
Look like it's purposely triggered when the conference_left event happens in jitsi, most likely to clear the redux state object between conferences, so it's not a bug.

Reply:
Ah, got it, thanks for explaining, that makes sense!